docs/docs.rst

```diff
@@ -10,4 +10,29 @@ You can build the docs by installing ``Sphinx`` and running::
 	# in the docs directory
 	make html
 
-Let us know if you have any questions or want to contribute to the documentation.
+Please follow these guidelines when updating our docs.
+Let us know if you have any questions or if something isn't clear.
+
+The brand
+---------
+
+We are called **Read the Docs**.
+The *the* is not capitalized.
+
+We do however use the acronym **RTD**.
+
+Titles
+------
+
+For page titles, or Heading1 as they are sometimes called, we use title-case.
+
+If the page includes multiple sub-headings (H2, H3),
+we usually use sentence-case unless the titles include terminology that is supposed to be capitalized.
+
+Content
+-------
+
+* Do not break the content across multiple lines at 80 characters,
+  but rather broken them on semantic meaning (e.g. periods or commas).
```

Comment:
_break_ them

Reply:
Also, I think it would be awesome to link the post that Eric shared with you. You said that was useful to understand this idea, so may be good to link it here.